test/e2e_node/container_lifecycle_test.go

```diff
@@ -753,7 +753,7 @@ var _ = SIGDescribe(framework.WithNodeConformance(), "Containers Lifecycle", fun
 		framework.ExpectNoError(results.Exits(regular1))
 	})
 
-	ginkgo.When("a pod is terminating because its liveness probe fails", func() {
+	ginkgo.When("a pod is terminating because ActiveDeadlineSeconds expires", func() {
```

Comment:
why do we modify an existing testcase instead of adding a new one?

We need to assert existing behavior does not change

Reply:
@hshiina please, address above review comment, thanks!

Reply:
Sorry. I added new tests instead of modifying.